docs/generated/packages/expo/generators/application.json

```
@@ -19,7 +19,7 @@
     "type": "object",
     "properties": {
       "directory": {
-        "description": "The directory of the new application.",
+        "description": "Full path to the application folder. When using positional name argument, this path should include the application name as the final segment.",
```

Would this be better to just include an example? 
```suggestion
        "description": "The directory of the new application. (e.g." /path/to/new-app"),
```

---

I thought about it, but then wanted to keep it as short as possible. I need to do some testing how LLMs pick this new description up. And yeah in case I can add an example. The crucial part there is whether the positional name argument is used or not, so you'd need to include that in the example almost.